core/vm/instructions.go

```diff
@@ -628,16 +628,8 @@ func opCreate(pc *uint64, interpreter *EVMInterpreter, scope *ScopeContext) ([]b
 	if interpreter.readOnly {
 		return nil, ErrWriteProtection
 	}
-	var (
-		value        = scope.Stack.pop()
-		offset, size = scope.Stack.pop(), scope.Stack.pop()
-		input        = scope.Memory.GetCopy(int64(offset.Uint64()), int64(size.Uint64()))
-		gas          = scope.Contract.Gas
-	)
-	if interpreter.evm.chainRules.IsEIP150 {
-		gas -= gas / 64
-	}
 
+	value := scope.Stack.pop()
 	if interpreter.evm.chainRules.IsEIP4762 {
```

Comment:
This should happen before L644, so that L653 never returns `false` (which is OK to ignore under normal circumstances, since `gas` will be `contract.Gas` or `63/64` of that.

Reply:
interesting, it was fixed in `CREATE2` but not `CREATE`... looks like a rebase issue to me.

But it's also incorrect in `CREATE2` because the gas is saved before the witness costs are charged.

I have to think what the correct fix is